File: XOWeek/iOS/iOS/ViewController.m
//
//  ViewController.m
//  iOS
//
//  Created by Ione Souza Junior on 20/08/17.
//  Copyright © 2017 Ione Souza Junior. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UILabel *lbDescricao;
- (IBAction)btCarregar:(UIButton *)sender;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)btCarregar:(UIButton *)sender {
    _lbDescricao.text = @"XOWeek - iOS";
}
@end

File: XOWeek/XamarinTradicional/iOS/ViewController.cs
using Foundation;
using System;
using UIKit;
using Core;

namespace iOS
{
    public partial class ViewController : UIViewController
    {
        private Evento _evento;

        public ViewController (IntPtr handle) : base (handle)
        {
            _evento = new Evento();
        }

        public override void ViewDidLoad()
        {
            base.ViewDidLoad();
        }

        public override void DidReceiveMemoryWarning()
        {
            base.DidReceiveMemoryWarning();
        }

        partial void btCarregar(UIButton sender)
        {
            lbDescricao.Text = _evento.GetInfo();
        }
    }
}